### PR TITLE
Enable TypeScript for Storybook

### PR DESCRIPTION
**Pull request by GRardB**

### CODE REVIEW

**Comment by GRardB** [comment]
[Get Storybook working with TypeScript](https://trello.com/c/mP14XRf6/26-get-storybook-working-with-typescript)

File: .storybook/config.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { configure } from '@storybook/react';
+import { configure } from '@storybook/react'
 
 // automatically import all files ending in *.stories.js
```
Comment (malinatran) [inline .storybook/config.js]: Remove comment.

File: .storybook/config.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
-const req = require.context('../stories', true, /.stories.js$/);
+const req = require.context('../stories', true, /.stories.tsx$/)
```
Comment (malinatran) [inline .storybook/config.js]: Delete comment above.

Comment (GRardB) [reply .storybook/config.js]: ¿Por qué?

**Pull request merged**